src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/MethodologySummary.test.tsx

```diff
@@ -345,6 +351,125 @@ describe('MethodologySummary', () => {
       ).toBeInTheDocument();
     });
 
+    test('the edit button is shown when a user can approve the methodology', () => {
+      render(
+        <MemoryRouter>
+          <MethodologySummary
+            publication={{
+              ...testPublicationNoMethodology,
+              methodologies: [
+                {
+                  ...testMethodology,
+                  permissions: {
+                    ...testMethodology.permissions,
+                    canApproveMethodology: true,
+                  },
+                },
+              ],
+            }}
+            topicId={testTopicId}
+            onChangePublication={noop}
+          />
+        </MemoryRouter>,
+      );
+
+      expect(
+        screen.getByText('Edit this methodology', { selector: 'a' }),
```

Comment:
Would suggest using the `getByRole` selector instead. We can use it like this:

```tsx
screen.getByRole('link', { name: 'Edit this methodology' });
```

This is a nicer to debug with as it tells you what the accessible elements are on the page and also reads a bit nicer as well. These are the elements  you're normally concerned with interacting with as a user.

Would apply the same selector to the other test cases too

Reply:
Thanks, I see the benefit in the log! I have made these changes throughout now.